README.md

```diff
@@ -16,15 +17,15 @@ allprojects {
 then, in your app `build.gradle`
 ```groovy
 dependencies {
-    compile "com.github.Commit451.Regalia:regalia:latest.version.here"
+    implementation 'com.github.CraZyLegenD.Regalia:latest-version'
     //for gson support
-    compile "com.github.Commit451.Regalia:regalia-gson:latest.version.here"
+    implementation 'com.github.CraZyLegenD.Regalia:regalia-gson:latest-version'
     //for LoganSquare support
-    compile "com.github.Commit451.Regalia:regalia-logansquare:latest.version.here"
+    implementation 'com.github.CraZyLegenD.Regalia:regalia-logansquare:latest-version'
     //for Moshi support
-    compile "com.github.Commit451.Regalia:regalia-moshi:latest.version.here"
+    implementation 'com.github.CraZyLegenD.Regalia:regalia-moshi:latest-version'
     //for Parceler support
-    compile "com.github.Commit451.Regalia:regalia-parceler:latest.version.here"
+    implementation 'com.github.CraZyLegenD.Regalia:regalia-parceler:latest-version'
```

Comment:
Change these

Reply:
I mean parceler isn't needed in Kotlin anymore, might aswell remove the whole module

Reply:
I was referring to the change from `com.github.Commit451` to `com.github.CraZyLegenD`